### DISCUSSION TITLE
About accuracy

### DISCUSSION THREAD

**Discussion by mshmoon** [Universite-Gustave-Eiffel/NoiseModelling#625] [Q&A]
May I ask if the author has compared the accuracy of noise modeling with other noise software on the market?

**Comment by pierromond** [comment]
@IsotoCedex did a nice work about it. Maybe he can share his work with you ?

**Reply by IsotoCedex** [reply]
Hi @mshmoon,
You can see the work to which @pierromond refers, in the following link.
https://documentacion.sea-acustica.es/publicaciones/Cuenca23/Abs_46.pdf
I'm sorry it's not in English. It was for a Spanish-Portuguese Congress.
